src/test/resources/regressions/features/termination/non-terminated_functions.gobra

```diff
@@ -0,0 +1,128 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package termination
+
+decreases x
+func f_e(x int) int {
```

Comment:
As an observation, function names in Go usually follow the camel case format (https://golang.org/doc/effective_go#names)